### DISCUSSION TITLE
Button 默认的 fontweight依然是加粗的？

### DISCUSSION THREAD

**Discussion by howzew** [irihitech/Semi.Avalonia#472] [Q&A] [answered]
已经更新至11.2.1，Button 默认的 fontweight依然是加粗的
想问一下，是就这么设计的，还是bug
谢谢。

**Comment by zdpcdt** [comment, marked as answer]
参考 [Semi Design](https://semi.design/zh-CN/basic/button)，就是这么设计的，Button的默认FontWeight就是600。

<img width="1143" alt="image" src="https://github.com/user-attachments/assets/78b0aa13-aa0d-4d75-a030-1d766a296cce">
